frontend/src/buttons/LaunchButton/LaunchButton.tsx

```diff
@@ -92,6 +92,10 @@ export const LaunchButton: React.FC<Props> = ({ connection, service, menuItem, s
               })
             : (launch = false)
           break
+        case 'HTTP':
+        case 'HTTPS':
+          window.open(`${service.type}://${app.connection?.host}:${app.connection?.port}`)
```

Comment:
The launch url is all contained in app.command I believe.
```suggestion
          launch = true
```

Reply:
launch = false